deploy/4_l1_transfer_tool.ts

```diff
@@ -0,0 +1,79 @@
+import consola from 'consola'
+import { utils } from 'ethers'
+
+import '@nomiclabs/hardhat-ethers'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+
+import { getDeploymentName, promptContractAddress } from './lib/utils'
+import { ethers, upgrades } from 'hardhat'
+import { L1GraphTokenLockTransferTool } from '../build/typechain/contracts/L1GraphTokenLockTransferTool'
+import path from 'path'
+import { Artifacts } from 'hardhat/internal/artifacts'
+
+const logger = consola.create({})
+
+const ARTIFACTS_PATH = path.resolve('build/artifacts')
+const artifacts = new Artifacts(ARTIFACTS_PATH)
+const l1TransferToolAbi = artifacts.readArtifactSync('L1GraphTokenLockTransferTool').abi
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployer } = await hre.getNamedAccounts()
+
+  // Get the addresses we will use
+  const tokenAddress = await promptContractAddress('L1 GRT', logger)
+  if (!tokenAddress) {
+    logger.warn('No token address provided')
+    process.exit(1)
+  }
```

Comment:
nit but the prompt and the check for non null address could be wrapped into a function to tidy up a bit.

Reply:
hmm yeah it might be a bit cleaner but I'd rather leave it like this for now